prime-select

```diff
@@ -126,13 +135,13 @@ class Switcher(object):
 
         return True
 
-    def _disable_nvidia(self):
+    def _disable_nvidia(self, keep_cuda=False):
```

Comment:
Maybe "keep_cuda" could become "keep_nvidia_modules"?

```suggestion
    def _disable_nvidia(self, keep_nvidia_modules=False):
```

Reply:
got it , will refine it.